worker/src/lib/HeliconeHeaders.ts

```diff
@@ -150,13 +150,21 @@ export class HeliconeHeaders implements IHeliconeHeaders {
     orgId?: string;
   }> {
     const heliconeAuth = this.headers.get("helicone-auth");
-    const heliconeAuthJWT = this.headers.get("helicone-jwt");
+
     if (heliconeAuth) {
       return {
         _type: "bearer",
-        token: heliconeAuth.replace("Bearer ", ""),
+        token: heliconeAuth,
       };
     }
+    const heliconeAuthFallback = this.headers.get("authorization");
```

Comment:
Why would this be the fallback? Is this only for api routes?

Reply:
yep